webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -239,14 +241,17 @@ export const GlideTable: React.FC<GlideTableProps> = ({
       if (row === data.length) return;
       // avoid showing 'empty rows' below data
       if (!data[row]) return;
+
+      const hoverStyle = row === hoveredRow ? { bgCell: '#323335' } : {}; // for some reason, using the consts from our reusable theme doesn't work...
```

Comment:
need to at least use a different color for light theme.

what approach were you trying that wasn't working? not sure if there's a preferred way of doing this but it looks like `getCssVar` would work?